build/karma.conf.js

```diff
@@ -61,7 +61,7 @@ module.exports = function (config) {
             { pattern: 'node_modules/prismjs/components/prism-typescript.min.js', included: true, watched: false}
         ],
 
-        exclude: [],
+        exclude: [ 'node_modules/**/*spec.js' ],
```

Comment:
Since we include other js files from `node_modules` in karma, making it `node_modules/**` overrides those include patterns declared earlier, and the tests won't be able to run. I had to introduce this because now we include `@angular/compiler-cli` in our project for AoT, and one of their spec files, will throw an error if we don't exclude (see: https://github.com/angular/angular/issues/13115). I think this is generic and safe enough. Let me know if you still disagree.